client/components/create-event.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+export default function Event() {
+  const { register, handleSubmit, formState: { errors } } = useForm();
+
+  const onSubmit = async data => {
+    try {
+      await fetch('/api/events', {
+        method: 'POST',
+        body: JSON.stringify(data),
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      });
+    } catch (error) {
+      console.error('Error:', error);
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <div>
+        <label className='text-white'>Event Name
+          <div>
+            <input type='text' className='text-black border rounded border-black bg-green-300' {...register('name', {
+              required: 'This is required.',
+              minLength: {
+                value: 4,
+                message: 'Minimium event name is 4'
+              },
+              maxLength: {
+                value: 10,
+                message: 'Maximum event name is 10'
+              }
+            })} />
+          </div>
+          <div>
+            <p>{errors.name?.message}</p>
```

Comment:
You probably don't want the `<p>` tag unless there is actually an error to display.